packages/sdk/src/evm/contracts/prebuilt-implementations/vote.ts

```diff
@@ -319,7 +319,7 @@ export class Vote implements UpdateableNetwork {
       this.contractWrapper.readContract.votingDelay(),
       this.contractWrapper.readContract.votingPeriod(),
       this.contractWrapper.readContract.token(),
-      this.contractWrapper.readContract.quorumNumerator(),
+      this.contractWrapper.readContract["quorumNumerator()"],
```

Comment:
I think this will break this function

Shouldn't it be `this.contractWrapper.readContract["quorumNumerator()"]()` (you're missing the parentheses at the end to call the function)

Reply:
great catch thank you!